great_tables/gt.py

```diff
@@ -406,7 +404,7 @@ def _render_as_html(
         return finalized_table
 
     def _finalize_html_table(
-        style: str, quarto_disable_processing: str, quarto_use_bootstrap: str, *args: Any
+        self, style: str, quarto_disable_processing: str, quarto_use_bootstrap: str, *args: Any
```

Comment:
cc @rich-iannone If this is unused, I think we can take it out (but can also punt that to a later PR :)

Reply:
Yes, this is entirely used. Let's remove it in this PR.